climada/util/yearsets.py

```diff
@@ -82,7 +82,6 @@ def impact_yearset(imp, sampled_years, lam=None, correction_fac=True, seed=None)
 
     #save calculations in yimp
     yimp.event_id = np.arange(1, n_sampled_years+1)
-    yimp.tag['yimp object'] = True
```

Comment:
Does this potentially break a functionality? This is one of the rare instances of a possibly relevant variable to be stored in the tags object.

Reply:
Not sure 😅 All tests pass on my end, so I don't think it does? 🤷

Reply:
I quickly checked and don't think it would break any functionality.